### PR TITLE
FourSuggestions

### PR DESCRIPTION
**Pull request by jotasprout**
Best API tutorial I’ve found, two desktop GitHub clients, vimtutor
command.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ This is my list of things that I'd like to learn. Mostly this will relate to sof
 - [ ] [Nginx](https://www.nginx.com)
 - [ ] [Let's Encrypt](https://letsencrypt.org/)
 - [ ] [Bash Shell](http://www.bash.academy/)
+- [ ] [Vim] (https://robots.thoughtbot.com/the-vim-learning-curve-is-a-myth)
+- [ ] [GitHub Desktop] (https://desktop.github.com/)
+- [ ] [GitKraken] (https://www.gitkraken.com/)
```
Comment (xeon-zolt) [inline README.md:21]: my fav and supports almost all desktop OS

File: README.md [hunk 2]
```diff
@@ -16,6 +16,9 @@ This is my list of things that I'd like to learn. Mostly this will relate to sof
 - [ ] [Nginx](https://www.nginx.com)
 - [ ] [Let's Encrypt](https://letsencrypt.org/)
 - [ ] [Bash Shell](http://www.bash.academy/)
+- [ ] [Vim] (https://robots.thoughtbot.com/the-vim-learning-curve-is-a-myth)
```
Comment (xeon-zolt) [inline README.md:19]: emac people are gonna be angry if they see only vim

**Comment by mattjegan** [comment]
Thanks @jotasprout

**Comment by jotasprout** [comment]
Thank you, @mattjegan.

**Pull request merged**